test/modules/offload-s3/offload_test.go

```diff
@@ -460,3 +460,167 @@ func Test_AutoTenantActivation(t *testing.T) {
 		}
 	})
 }
+
+func Test_ConcurrentFreezeUnfreeze(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Minute)
```

Comment:
yeah it I blame copy/past 😅 , but i agree it will be timeout automatically by the CI if crossed ., wdyt if will replace it later with another PR to not re-trigger the pipeline